package.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<package format="1" xmlns="https://wiki.freecad.org/Package_Metadata">
+  <name>Volmdlr Workbench</name> <!-- What the Addon Manager displays to users -->
+  <description>A computations-oriented python VOLume MoDeLeR with STEP support for import and export.</description>
+  <version>0.9.0</version> <!-- Semantic versioning (1.2.3-beta) or CalVer-based, (2022.01.07), don't omit or non-git installations won't see your updates -->
+  <date>2023-03-06</date> <!-- Date of the last update to the version number -->
+  <maintainer email="your_address@null.com">Your Name</maintainer>
+  <license file="LICENSE">GPL-3</license> <!-- Make sure you actually have this file in your Addon repo if the license requires it -->
+  <url type="repository" branch="master">https://github.com/Dessia-tech/volmdlr</url> <!-- Don't forget to update the branch name here -->
+  <url type="readme">https://github.com/Dessia-tech/volmdlr/blob/master/README.md</url> <!-- Link to the HTML-rendered README page -->
+  <icon>path/to/icon.svg</icon> <!-- If you include your icon here, you don't have to submit it to the main FreeCAD repo -->
```

Comment:
Is it possible for this workbench to have an icon that is in SVG format, it makes scaling seamless on HiDPi screens and therefore preferable to display in the Addon Manager (and in the workbench in general)

Reply:
We don't have an icon for the moment, I keep this in mind